django/db/backends/postgresql/introspection.py

```diff
@@ -211,11 +215,19 @@ def get_constraints(self, cursor, table_name):
             [table_name],
         )
         for constraint, columns, kind, used_cols, options in cursor.fetchall():
+            foreign_key = None
+            if kind == "f":
+                foreign_key = tuple(tuple(col.split(".", 1)) for col in used_cols)
+                # If the foreign key has a single column, return a single tuple (for
+                # backwards compatibility).
```

Comment:
Is this public API? If not we can probably break backward compatibility here.

Reply:
I'm not sure what is considered a public API exactly, this is introspection so maybe it's not public? Actually I could return a single tuple for composite fks in the format (table, col 1, col 2)

Reply:
I went with the above idea, keeping this a single tuple.